apiserver/README.md

```diff
@@ -28,39 +27,75 @@ Make sure the version of Helm is v3+. Currently, [existing CI tests](https://git
 helm version
 ```
 
+### Install KubeRay Operator
+
+* Install a stable version via Helm repository (only supports KubeRay v0.4.0+)
+
+  ```sh
+    # Install the KubeRay helm repo
+    helm repo add kuberay https://ray-project.github.io/kuberay-helm/
+
+    # Install KubeRay Operator v0.6.0.
+    helm install kuberay-operator kuberay/kuberay-operator --version 0.6.0
+
+    # Check the KubeRay Operator Pod in `default` namespace
+    kubectl get pods
+    # NAME                                             READY   STATUS    RESTARTS   AGE
+    # kuberay-operator-7456c6b69b-t6pt7                1/1     Running   0          172m
+
+  ```
+
 ### Install KubeRay APIServer
 
 * Install a stable version via Helm repository (only supports KubeRay v0.4.0+)
+  
   ```sh
+  # Install the KubeRay helm repo
   helm repo add kuberay https://ray-project.github.io/kuberay-helm/
 
   # Install KubeRay APIServer v0.6.0.
-  helm install kuberay-apiserver kuberay/kuberay-apiserver --version 0.6.0
+  helm install -n ray-system kuberay-apiserver kuberay/kuberay-apiserver --version 0.6.0
```

Comment:
Does it have to be installed in the `ray-system` namespace? (especially since the operator is in the `default` namespace. Shouldn't we have both installed side by side?

Reply:
Fixed in [a16b6d3](https://github.com/ray-project/kuberay/pull/1435/commits/a16b6d3e7efb66ec43e34c78090d9a1692f3c885)